src/poprox_recommender/default.py

```diff
@@ -82,23 +85,31 @@ def personalized_pipeline(num_slots: int, algo_params: dict[str, Any] | None = N
         logger.info("Recommendations will be ranked with plain top-k.")
         ranker = topk_ranker
 
-    pipeline = RecommendationPipeline(name=diversify)
+    # TODO put pipeline name back in
```

Comment:
Worth resolving this now or want to wait for a follow-up PR?

Reply:
#85 adds it :)